libs/data/mlpe/data/transforms/preprocessor.py

```diff
@@ -17,32 +15,43 @@ class Preprocessor(torch.nn.Module):
     will normalize both strain and parameters.
 
     Args:
-        normalizer: Callable that normalizes parameters. If None,
-        will return parameters as is.
+        num_ifos:
+            Number of interferometers.
+        sample_rate:
+            Sample rate of the data.
+        fduration:
+            Duration of the time domain whitening filter.
+        normalizer:
+            torch.nn.Module that takes in a tensor of parameters
+            of shape (n_dim, batch) and returns a transformed
+            tensor of shape (n_dim, batch). If not passed, the identity
+            transform is performed.
     """
 
     def __init__(
         self,
         num_ifos: int,
         sample_rate: float,
-        kernel_length: float,
-        normalizer: Optional["Transform"] = None,
         fduration: Optional[float] = None,
-        highpass: Optional[float] = None,
+        normalizer: torch.nn.Module = torch.nn.Identity(),
```

Comment:
Why is the normalizer default identity?

Reply:
So that it will just return the parameters themselves if a normalizer isn't passed

Reply:
Minor nitpick. But can we call it something different. Normalizer sounds like it will always normalize. Maybe `scaler`?

Reply:
Sure